homeassistant/components/holiday/calendar.py

```diff
@@ -63,10 +74,11 @@ async def async_setup_entry(
     """Set up the Holiday Calendar config entry."""
     country: str = config_entry.data[CONF_COUNTRY]
     province: str | None = config_entry.data.get(CONF_PROVINCE)
+    categories: list[str] | None = config_entry.options.get(CONF_CATEGORIES)
```

Comment:
Here you are reading the province from `entry.data`, and the categories from `entry.options`, but I understood that they were both moved to options instead.